monkey/agent_plugins/payloads/ransomware/src/ransomware_options.py

```diff
@@ -0,0 +1,63 @@
+import re
+
+from pydantic import Field, validator
+
+from common.base_models import InfectionMonkeyBaseModel
+
+from .options_validators import (
+    VALID_FILE_EXTENSION_REGEX,
+    VALID_RANSOMWARE_PATH_LINUX_REGEX,
+    VALID_RANSOMWARE_PATH_WINDOWS_REGEX,
+)
+
+
+class EncryptionBehavior(InfectionMonkeyBaseModel):
+    file_extension: str = Field(
+        default=".m0nk3y",
+        description="The file extension that the Infection Monkey will use"
+        " for the encrypted file.",
+    )
+    linux_target_dir: str = Field(
+        default="",
+        description="A path to a directory on Linux systems that contains"
+        " files you will allow Infection Monkey to encrypt. If no"
+        " directory is specified, no files will be encrypted.",
+    )
+    windows_target_dir: str = Field(
+        default="",
+        description="A path to a directory on Windows systems that contains"
+        " files you will allow Infection Monkey to encrypt. If no"
+        " directory is specified, no files will be encrypted.",
+    )
+
+    @validator("file_extension")
+    def validate_file_extension(cls, file_extension):
+        if not re.match(VALID_FILE_EXTENSION_REGEX, file_extension):
+            raise ValueError("Invalid file extension provided")
+        return file_extension
+
+    @validator("linux_target_dir")
+    def validate_linux_target_dir(cls, linux_target_dir):
+        if not re.match(VALID_RANSOMWARE_PATH_LINUX_REGEX, linux_target_dir):
+            raise ValueError("Invalid Linux target directory provided")
+        return linux_target_dir
+
+    @validator("windows_target_dir")
+    def validate_windows_target_dir(cls, windows_target_dir):
+        if not re.match(VALID_RANSOMWARE_PATH_WINDOWS_REGEX, windows_target_dir):
+            raise ValueError("Invalid Windows target directory provided")
+        return windows_target_dir
+
+
+class OtherBehaviors(InfectionMonkeyBaseModel):
+    pass
+
```

Comment:
I don't think we need this any longer. This was driven by a desire for the UI to look a certain way.